### DISCUSSION TITLE
form - text input width to end of screen

### DISCUSSION THREAD

**Discussion by dncpax** [Budibase/budibase#2951] [Help] [answered]
Hi there.

I've figured out how to make an horizontal form using a container.
But I can't figure out how to have the final text input go all the way to the right end of the screen?
If I make it a fixed width it will break in mobile screens...

Thanks.

**Comment by mjashanks** [comment]
Hi @dncpax 

The trick is ...
1. Put your final field inside it's own container
2. Set that container to `Grow`

https://user-images.githubusercontent.com/3524181/136782952-55a00d04-60d6-4185-a979-2876bafff420.mp4

**Comment by dncpax** [comment, marked as answer]
ahah wicked!
